pkg/cli/schemaherokubectlcli/approve_migration.go

```diff
@@ -24,7 +24,10 @@ func ApproveMigrationCmd() *cobra.Command {
 		SilenceErrors: true,
 		SilenceUsage:  true,
 		PreRun: func(cmd *cobra.Command, args []string) {
-			viper.BindPFlags(cmd.Flags())
+			err := viper.BindPFlags(cmd.Flags())
```

Comment:
Thanks for the contribution! I think we should handle these errors instead of just printing them. I believe that there's a `PreRunE` version of this function, where we can return an error preventing the function execution from continuing. Would you be up for swapping this out for the `PreRunE` and then `return err` instead?

Reply:
@marccampbell thanks but i did not get what to do , can you help me please? thanks